Docs/source/install/hpc.rst

```diff
@@ -52,6 +52,7 @@ This section documents quick-start guides for a selection of supercomputers that
    hpc/summit
    hpc/taurus
    hpc/greatlakes
+   hpc/pitzer
```

Comment:
Please add in alphabetic order in this table of content.

(Yes, `greatlakes` is also off :sweat_smile: )